### PR TITLE
add package py-numpydoc

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-numpydoc/package.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+class PyNumpydoc(PythonPackage):
+    """numpydoc - Numpy's Sphinx extensions"""
+
+    homepage = "https://github.com/numpy/numpydoc"
+    url      = "https://pypi.io/packages/source/n/numpydoc/numpydoc-0.6.0.tar.gz"
+
+    version('0.6.0', '5f1763c44e613850d56ba1b1cf1cb146')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numpydoc/package.py]: Might want to add:
```python
depends_on('python@2.6:2.8,3.3:')
```
from the `setup.py`:
```python
if sys.version_info[:2] < (2, 6) or (3, 0) <= sys.version_info[0:2] < (3, 3):   
    raise RuntimeError("Python version 2.6, 2.7 or >= 3.3 required.")
```

File: var/spack/repos/builtin/packages/py-numpydoc/package.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    depends_on('py-setuptools',    type='build')
+    depends_on('py-sphinx@1.0.1:', type='build')
+    depends_on('py-jinja2@2.3:',   type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numpydoc/package.py]: I'm not seeing where the `jinja2` dependency is specified. Are you sure neither of these are `run` dependencies?

Comment (healther) [inline var/spack/repos/builtin/packages/py-numpydoc/package.py]: from ```setup.py```
    install_requires=["sphinx >= 1.0.1", 'Jinja2>=2.3'],

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numpydoc/package.py]: From which `setup.py`? I'm not seeing that in the tarball from this PR. All I see is:
```python
requires=["sphinx (>= 1.0.1)"],
```
Maybe you're looking at a different version?

Comment (healther) [inline var/spack/repos/builtin/packages/py-numpydoc/package.py]: you're right, I was looking at the current master of its git-repo. Thanks

**Pull request merged**